openml-caret/README.md

```diff
@@ -0,0 +1,21 @@
+OpenML Provider for caret models
+------------------------------------------------------------
+
+This module contains an OpenML provider for loading custom code that implements
```

Comment:
this doesn't load custom code that implements Feedzai's Caret API; this module contains an OpenML (link to feedzai openml repository?) provider for loading models trained with the Caret package (link to Caret: http://caret.r-forge.r-project.org/)